draft-ietf-acme-acme.md

```diff
@@ -955,18 +954,17 @@ key but not the corresponding registration URI to recover the registration URI.
 
 If the server wishes to present the client with terms under which the ACME
 service is to be used, it MUST indicate the URI where such terms can be accessed
-in a Link header with link relation "terms-of-service".  As noted above, the
-client may indicate its agreement with these terms by updating its registration
-to include the "agreement" field, with the terms URI as its value.  When these
-terms change in a way that requires an agreement update, the server MUST
-use a different URI in the Link header.
+in the "terms-of-service" subfield of the "meta" field in the directory object,
+and the server MUST reject new-registration requests that do not contain
```

Comment:
Again, I'm trying to simplify the protocol here. There is zero reason for the protocol to support a server that offers a ToS URL but treats them as optional for registering an account. This only creates confusion.